en/firefox/faq.ftl

```diff
@@ -0,0 +1,99 @@
+
+not-only-is = Not only is { -brand-name-firefox } safe to use, it also helps keep your data and private information safe. The { -brand-name-firefox-browser } automatically blocks known third party trackers, social media trackers, cryptominers and fingerprinters from collecting your data. <a href="{ $url }">Learn more about the privacy in our products</a>.
+
+related-questions-safe = Related questions: is { -brand-name-firefox } good for privacy, is { -brand-name-firefox } secure, Is { -brand-name-firefox } better for privacy
+does-firefox-sell = Does { -brand-name-firefox } sell your personal data?
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/privacy/
+
+nope-never-have = Nope. Never have, never will. And we protect you from many of the advertisers who do. { -brand-name-firefox } products are designed to protect your privacy. <a href="{ $url }">That’s a promise.</a>
+
+why-is-firefox = Why is { -brand-name-firefox } so slow?
+firefox-isnt-slow = { -brand-name-firefox } isn’t slow… now. In 2017, we completely rebuilt our browser engine (called Quantum), to ensure { -brand-name-firefox } could compete with other major browsers. And, our tracker blockers help pages load even faster. So { -brand-name-firefox } is lightning fast without sacrificing any of your privacy.
+
+is-firefox-chromium = Is { -brand-name-firefox } { -brand-name-chromium } based?
+firefox-is-not = { -brand-name-firefox } is not based on { -brand-name-chromium } (the open source browser project at the core of { -brand-name-google } { -brand-name-chrome }). In fact, we’re one of the last major browsers that isn’t. { -brand-name-firefox } runs on our Quantum browser engine built specifically for { -brand-name-firefox }, so we can ensure your data is handled respectfully and kept private.
+
+does-firefox-use = Does { -brand-name-firefox } use { -brand-name-google }?
+
+# Variables:
+# $url (url) - link to https://support.mozilla.org/kb/change-your-default-search-settings-firefox
+
+firefoxs-default-search = { -brand-name-firefox }’s default search engine is { -brand-name-google }, which means you can search the web directly from the address bar. <a href="{ $url }">Learn more about search engine preferences and changing defaults.</a>
+
+does-firefox-have = Does { -brand-name-firefox } have a built-in VPN?
+
+# Variables:
+# $url (url) - link to https://fpn.firefox.com/
+# $url2 (url) - link to https://vpn.mozilla.org/
+
+firefox-does-not = { -brand-name-firefox } does not have a built-in VPN (virtual private network), but there are two products made by { -brand-name-mozilla }/{ -brand-name-firefox } that you can use in addition to the private { -brand-name-firefox-browser } that can protect either your browser (<a href="{ $url }">{ -brand-name-firefox-private-network }</a>) or device (<a href="{ $url2 }">{ -brand-name-mozilla-vpn }</a>) connections on WiFi, as well as your IP address.
```

Comment:
Should it be singular in `connection on WiFi` as in *either (singular) ABC or (singular) XYZ*?